app/controllers/books_controller.rb

```diff
@@ -0,0 +1,10 @@
+class BooksController < ApplicationController
+    def index
```

Comment:
**[Minor]** The indentation of your files is mixing 2 vs. 4 spaces - I think it's mostly 2 for files that were generated (e.g., config files, specs, etc.) and 4 for the ones you created yourself, maybe. 😅 Internally we're used to use 2 spaces, as it leaves a bit more room for actual code, we actually have [rubocop](https://github.com/rubocop/rubocop) enforcing the that layout consistency rule. Not a big deal for now, though!

Reply:
Yes, I noticed that and was too lazy to change my computer's VS settings :upside_down_face:   (it's done now!)
I also used rubocop in my thesis internship before, so that's why I never felt the need to change the settings :smile: